f5_openstack_agent/lbaasv2/drivers/bigip/test/test_exceptions.py

```diff
@@ -0,0 +1,533 @@
+#!/usr/bin/env python
+# Copyright 2017 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import inspect
+import logging
+import pdb
+import pytest
+import sys
+import traceback
+
+import f5_openstack_agent.lbaasv2.drivers.bigip.exceptions as exceptions
+
+from collections import namedtuple
+from mock import Mock
+from mock import mock_open
+from mock import patch
+
+
+class Test_F5MissingDependencies(object):
+    def setup_traceback(self):
+        traceback.print_exc = Mock()
```

Comment:
Let's not use `setup_traceback` just assign `traceback.print_exc = Mock()` within the test.